editor/src/components/custom-code/code-file.ts

```diff
@@ -144,6 +144,7 @@ export function componentDescriptor(
 
 export interface ComponentDescriptorWithName extends ComponentDescriptor {
   componentName: string
+  moduleName: string
```

Comment:
for components that exist in the project, is this some sort of normalized path? If yes, I would put this fact in the name!